src/Codeception/Module/Symfony2.php

```diff
@@ -358,8 +358,17 @@ public function seeCurrentRouteIs($routeName, array $params = [])
         if (!$router->getRouteCollection()->get($routeName)) {
             $this->fail(sprintf('Route with name "%s" does not exists.', $routeName));
         }
-        $url = $router->generate($routeName, $params);
-        $this->seeCurrentUrlEquals($url);
+
+        try {
+            $match = $router->match(explode('?', $this->grabFromCurrentUrl())[0]);
```

Comment:
Is this overly simplistic if the route is defined with placeholders within query params?